Eyes/ImageTester/src/csharp/main/Applitools.ImageTester.Runner.csproj

```diff
@@ -37,7 +37,7 @@
     </Reference>
     <Reference Include="Eyes.Bundle, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\bin\Release\Eyes.Bundle.dll</HintPath>
+      <HintPath>..\bin\Debug\Eyes.Bundle.dll</HintPath>
```

Comment:
It breaks build for 'Release' configurtion. We should use '$(Configuration)' variable here instead